user-manual/administer/settings.rst

```diff
@@ -1871,3 +1872,132 @@ setting.
+Data Protection Regulation (GDPR). The GDPR came into effect on May 1, 2018, and
+even if your institution is not situtated in the EU, it is important to be aware
+of this regulation and how it might affect you.
+
+In compliance with the GDPR, AtoM now makes explicit its collection and use of
+cookies. AtoM collects cookies in order to enable browsing and loading of
+certain types of content. Visitors to AtoM sites who do not wish to have cookies
+placed on their computers should set their browsers to refuse cookies. However,
+certain features may not function properly without the aid of cookies.
+
+AtoM supports integration with `Google Analytics
+<https://www.google.com/analytics/>`_ for the purposes of gathering statistics on
+page views, site usage, user location, and other data on site visits. All data
+collected by Google Analytics are stored and processed by Google, according to
+the `Google Ads Data Processing Terms
+<https://privacy.google.com/businesses/processorterms/>`_
+
+None of the information gathered through the use of cookies or Google Analytics
+is used for any purpose other than the ones described here.
+
+System administrators will need to perform some manual steps to finalize the
+setup of the new features when upgrading or installing AtoM 2.4.1 for the first
+time.
+
+New 2.4.1 installations
+-----------------------
+
+Update cookie banner
+^^^^^^^^^^^^^^^^^^^^
+
+* The default cookie notification banner message will require updating.  This
+  can be done in |gears| *Admin > Settings > Privacy Notification*.
+* The default message will require the URL to the Privacy Policy static page be
+  updated. Currently the message reads:
+
+     .. code-block:: none
+
+        This website uses cookies to enhance your ability to browse and load
+        content. This website uses cookies to enhance your ability to browse
+        and load content. More Info: http://10.10.10.10/privacy
+
+Administrators can edit the text of the banner message, and choose whether or
+not to display the banner when a user accesses your site for the first time.
+Here's an example of how the notification banner will look:
+
+.. image:: images/gdpr-banner-post-set.*
+   :align: center
+   :width: 80%
+   :alt: GDPR banner first access
+
+* Update the URL in the message to the base URL for your site (see :ref:`Site
+  information <site-information>`)
+* If desired, you can use the current custom link syntax formatting to style the
+  hyperlink -  see :ref:`formatting` for more information
+
+Edit the Privacy Policy
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Administrators should also review the wording of the new default Privacy Policy
+static page. This can be found in |gears| *Admin > Static Pages >Privacy
+Policy*, or it can be accessed via the Quick links menu in the AtoM header.
+Click on ``edit`` at the bottom of the view page, make your desired edits, and
+select "save" to save your changes or ``cancel`` to return to the default text.
```

Comment:
Minor suggestion: add something like the following?

For more information on working with :term:`static pages <static page>` in AtoM, see: 

* :ref:`manage-static-pages`